usecases/findtags.go

```diff
@@ -38,13 +38,13 @@ func (s FindTagsUsecase) FindTags(cmd FindTagsCmd) (FindTagsResult, error) {
 		return FindTagsResult{}, errors.New("failed extracting paragraphs: " + err.Error())
 	}
 
-	rgx := regexp.MustCompile(strings.Join(cmd.Tags, "|"))
+	rgx := regexp.MustCompile("(?i)" + strings.Join(cmd.Tags, "|"))
```

Comment:
Could you please add a test to verify that the regex is doing what it is supposed to do?
https://blog.alexellis.io/golang-writing-unit-tests/

we could also have a short pairing session.